dvc/dependency/local.py

```diff
@@ -1,8 +1,53 @@
 from __future__ import unicode_literals
 
-from dvc.dependency.base import DependencyBase
+import os
+
+from dvc.pkg import Pkg
+
+from dvc.utils.compat import urlparse
+
+from dvc.dependency.base import DependencyBASE
 from dvc.output.local import OutputLOCAL
 
 
-class DependencyLOCAL(DependencyBase, OutputLOCAL):
-    pass
+class DependencyLOCAL(DependencyBASE, OutputLOCAL):
+    PARAM_PKG = "pkg"
+
+    def __init__(self, stage, info, remote=None):
+        pkg_info = info.pop(self.PARAM_PKG, None)
+
+        super(DependencyLOCAL, self).__init__(stage, info, remote=remote)
+
+        if not pkg_info:
+            return
+
+        self.pkg = Pkg(stage.repo.config, pkg_info)
+
+        if not self.pkg.installed:
+            self.pkg.install()
+
+        out_path = os.path.join(
+            self.pkg.repo.root_dir, urlparse(self.url).path.lstrip("/")
+        )
+
+        outs = self.pkg.repo.find_outs_by_path(out_path)
+        assert len(outs) == 1
```

Comment:
It should work with multiple outputs as well. See my comment to `pkg import` command.

Reply:
What do you mean? Something like
```
dvc pkg import pkg data1 data2 data3
```
and
```
dvc pkg import pkg data1 data2 data3 outputdir/
```
?